apps/weshipit/pages/consulting.tsx

```diff
@@ -29,51 +29,73 @@ export function Consulting() {
-        </Text>
+      <div className="relative">
+        <video
+          autoPlay
+          loop
+          muted
+          className="absolute left-0 top-0 z-0 h-full w-auto min-w-full max-w-none object-cover opacity-80"
+          style={{ zIndex: -1 }}
+        >
+          <source
+            src="https://player.vimeo.com/progressive_redirect/playback/849077469/rendition/1080p/file.mp4?loc=external&signature=4d9740bee6a0997e8358e6e039238313d6cf22ff06d6a5a0c6b537368a466e56"
+            type="video/mp4"
+          />
+          Your browser does not support the video tag.
+        </video>
+        <div className="relative z-10 h-screen">
+          <Hero>
+            <Text as="h1" variant="h1" className="mt-24 text-center uppercase">
+              <span className="tracking-widest text-black">
+                Straightforward
+              </span>{' '}
+              <br />
+              <small className="tracking-wide text-black">
+                React Native solutions
+              </small>
+            </Text>
 
-        <div className="mt-12 flex justify-center">
-          <Button
-            as="a"
-            href="https://cal.com/davidl/coaching"
-            isExternalLink={true}
-          >
-            Schedule one-on-one consulting
-          </Button>
-        </div>
-      </Hero>
+            <div className="mt-12 flex justify-center">
+              <Button
+                as="a"
+                href="https://cal.com/davidl/coaching"
+                isExternalLink={true}
+              >
+                Schedule one-on-one consulting
+              </Button>
+            </div>
+          </Hero>
 
-      <div className="mx-auto max-w-2xl p-3">
-        <Text as="p" variant="p1" className="mb-4">
-          Every week, I receive emails asking me for a moment with me. I never
-          know who to say yes to, who to say no to, and besides, my time is
-          precious, I don’t want to waste it. I have absolutely no desire to
-          devalue the most precious resource of my life: time.
-        </Text>
-        <Text as="p" variant="p1" className="mb-4">
-          The principle is simple: a 60-minute slot, online payment. You can
-          send me before the call if you wish a clear email in bullet points,
-          which summarizes the objectives of our conversation.
-        </Text>
-        <Text as="p" variant="p1">
-          My goal is for this hour to be most helpful to you.
-        </Text>
-        <Text as="p" variant="p1">
-          <em>
-            Note: If the calendar below doesn’t load in a few seconds, try
-            refreshing the page. Or you can{' '}
-            <Hyperlink href="https://cal.com/davidl/coaching">
-              open the bookling link on another tab
-            </Hyperlink>
-            .
-          </em>
-        </Text>
+          <div className="mx-auto max-w-2xl rounded-lg bg-slate-100 p-8">
+            <Text as="p" variant="p1Black" className="mb-4 ">
```

Comment:
Instead of adding a new variant `p1Black` —who does toooo many things at once size and color.
have you tired to add a `prose` wrapper? make it simple and consistent with website.